articles/cognitive-services/cognitive-services-support-options.md

```diff
@@ -78,7 +78,7 @@ The Cognitive Services release notes are updated as new releases are made availa
 
  - [Custom Vision](custom-vision-service/release-notes.md)
  - [Face](face/releasenotes.md)
- - [Language Understanding (LUIS)](luis/whats-new.md)
+ - [Language Understanding (LUIS)](LUIS/whats-new.md)
```

Comment:
This is case insensitive, so this change in unnecessary. Not blocking, just an FYI.

Reply:
@IEvangelist thanks. I was getting a 404 when I tried to access the LUIS Github page from Github, so not sure what was causing it. But looks like it's not there in the docs. Thanks again.